web/docs/enums/FlowDirection.md

```diff
@@ -6,12 +6,20 @@
 
 • **Ingress** = ``"0"``
 
-Incoming traffic, from node observation point
+Incoming traffic, from the node observation point
 
 ___
 
 ### Egress
 
 • **Egress** = ``"1"``
 
-Outgoing traffic, from node observation point
+Outgoing traffic, from the node observation point
+
+___
+
+### Inner
+
+• **Inner** = ``"2"``
+
+Inner traffic, with same source and destination node
```

Comment:
```suggestion
Inner traffic, with source and destination traffic in the same node
```
I think singular `node` is what we want here for sure. Alternatively, maybe just this: 

Inner traffic, with the same source and destination node